packages/expo-device/src/Device.types.ts

```diff
@@ -0,0 +1,289 @@
+export const devicesWithNotch = [
```

Comment:
These are constants not types

Reply:
made a new file `DeviceConstants.ts` to contain constants